### ISSUE TITLE
APT whitelist request for libmediainfo0

### ISSUE THREAD

**Issue by sbraz** [travis-ci/apt-package-whitelist#2918]

**Comment by sbraz** [comment]
Hi @BanzaiMan how long does it take for the whitelist to be propagated to the production build environment?
https://travis-ci.org/sbraz/pymediainfo/jobs/128236962

**Comment by BanzaiMan** [comment]
@sbraz The PR added libmediainfo0v5, which was found by examining the source package.

**Comment by BanzaiMan** [comment]
The current workflow may be deficient in this situation. I do not have a good answer for it.

**Comment by sbraz** [comment]
My bad, I didn't realise this was the full package name. It still fails with the right name though: https://travis-ci.org/sbraz/pymediainfo/jobs/128239921

**Comment by BanzaiMan** [comment]
Right. There are a few cases like this, when the originally requested package's source package does not contain the original package name. This is what I meant in the last commit.

I think I can manually allow this, but I intend to do it next week. Sorry for the delay.

**Comment by sbraz** [comment]
@BanzaiMan No hurry, thanks :) Can you re-open the issue so you won't forget?

**Comment by sbraz** [comment]
@BanzaiMan Hi, any chance you could re-open this?

**Comment by sbraz** [comment]
@BanzaiMan Hi again, sorry to bother you once again, do you think you'll have some time to look at it in the near future?

**Comment by BanzaiMan** [comment]
@sbraz I'm afraid not. If you need a particular package that isn't addressed by the whitelist, your best bet is to use `sudo: required`. You can either call `sudo apt-get install …` directly, or continue to use the addon (https://docs.travis-ci.com/user/installing-dependencies#Installing-Packages-with-the-APT-Addon). (That is to say, if you use the addon on a `sudo`-enabled infrastructure, the addon will bypass the whitelist to install any package.)

**Comment by sbraz** [comment]
Should I keep the issue opened in case you add it manually or should I close it now?
